### PR TITLE
Update changelog for 1.0.0-BETA13

### PR DESCRIPTION
**Pull request by jmikola**
Documenting features in the [1.0.0-BETA13](https://github.com/doctrine/mongodb-odm/issues?milestone=7&q=is%3Aclosed) milestone.

### CODE REVIEW

File: CHANGELOG-1.0.md [hunk 1]
```diff
@@ -15,6 +15,51 @@ To generate a changelog summary since the last version, run
 1.0.x-dev
 ---------
 
+1.0.0-BETA13 (2015-05-21)
+-------------------------
+
+All issues and pull requests in this release may be found under the
+[1.0.0-BETA13 milestone](https://github.com/doctrine/mongodb-odm/issues?milestone=7&q=is%3Aclosed).
+
+#### `atomicSet` and `atomicSetArray` stratgies for top-level collections
```
Comment (BernardoSilva) [inline CHANGELOG-1.0.md]: Typo: `stratgies `

Comment (jmikola) [inline CHANGELOG-1.0.md]: Thanks! Will fix :)

File: CHANGELOG-1.0.md [hunk 2]
```diff
@@ -15,6 +15,55 @@ To generate a changelog summary since the last version, run
 1.0.x-dev
 ---------
 
+1.0.0-BETA13 (2015-05-21)
+-------------------------
+
+All issues and pull requests in this release may be found under the
+[1.0.0-BETA13 milestone](https://github.com/doctrine/mongodb-odm/issues?milestone=7&q=is%3Aclosed).
```
Comment (malarzm) [inline CHANGELOG-1.0.md]: This link shows wrong results after going to 2nd page, https://github.com/doctrine/mongodb-odm/issues?q=milestone%3A1.0.0-BETA13+is%3Aclosed is better

**Comment by jmikola** [comment]
@malarzm: Ready for another review when you get a chance.

**Comment by malarzm** [comment]
LGTM! :+1:

**Comment by smolinari** [comment]
What does LGTM mean? Is it really Let's Get The Money? That's the only thing I could find with Google that made any sense. LOL! :smile: 

Also, when will the ODM be feature frozen as a beta and released as final?

Scott

**Comment by malarzm** [comment]
Heh I must admit that "Let's Get The Money" in context of release sounds awesome :) but we use that as "Looks Good To Me".

I'd say that as soon as we wrap up remaining bugs we are aware of (and these reported in the meantime) 1.0.0 shall be released.

**Comment by smolinari** [comment]
:+1:  Thanks. Looking forward to it.

Scott

**Pull request merged**